src/test/java/com/bytebreak/animagic/integration/TexturePacking.java

```diff
@@ -9,6 +9,10 @@ public static void main(String[] arg) {
         TexturePacker.Settings settings = new TexturePacker.Settings();
         settings.maxWidth *= 4;
         settings.maxHeight *= 4;
+        settings.combineSubdirectories = true;
+        settings.duplicatePadding = true;
+        settings.fast = true;
+        settings.square = true;
```

Comment:
These were just other settings I found that looked possibly useful. There is also cool built-in support for scaling graphics up. meaning we can draw them at any resolution we want and just have the texturepacker do the scaling.